terraform/modules/cloud/aws/deploy/fleet/templates/user_data.bash

```diff
@@ -1,51 +1,41 @@
 #!/bin/bash
-set -x
-exec > >(tee /tmp/user-data.log|logger -t user-data ) 2>&1
-
-#SETTING UP HOSTNAME
-
-REGION="${region}"
-IP=`curl -s http://169.254.169.254/latest/meta-data/public-ipv4`
-echo "aws-$REGION-$IP" > /etc/hostname
-hostname "aws-$REGION-$IP"
-
-#SETUP DATADOG
 
-MEM=$(awk '/MemTotal/ {printf("%.0f\n", $2 / 1024 / 1024)}' /proc/meminfo)
-FLAVOR="c$(nproc).m$MEM"
-DATADOG_API_KEY=`aws --region ${region} secretsmanager get-secret-value --secret-id datadog_api_key  | jq '.SecretString | fromjson | .datadog_api_key'`
-
-sed -i -- "s/DATADOG_API_KEY/$DATADOG_API_KEY/g" /etc/datadog-agent/datadog.yaml
-sed -i -- "s/region:unknown/region:${region}/g" /etc/datadog-agent/datadog.yaml
-sed -i -- "s/color:unknown/color:${color}/g" /etc/datadog-agent/datadog.yaml
-sed -i -- "s/env:unknown/env:${env}/g" /etc/datadog-agent/datadog.yaml
-sed -i -- "s/flavor:.*$/flavor:$FLAVOR/g" /etc/datadog-agent/datadog.yaml
-sed -i -- "s/127.0.0.1:3013/$IP:3013/g" /etc/datadog-agent/conf.d/http_check.d/conf.yaml
-
-sudo service datadog-agent start
-
-#INSTALL EPOCH
+# Don't enable debug mode in this script if secrets are managed as it ends-up in the logs
+# set -x
+exec > >(tee /tmp/user-data.log|logger -t user-data ) 2>&1
 
-URL="${epoch_package}"
+export INFRASTRUCTURE_ANSIBLE_VAULT_PASSWORD=`aws --region ${region} secretsmanager get-secret-value --secret-id ansible_vault_password --output text --query 'SecretString'`
 
-wget -q $URL -O /home/epoch/epoch.tar.gz
+git clone -b master --single-branch https://github.com/aeternity/infrastructure.git /infrastructure
+cd /infrastructure/ansible
 
-mkdir /home/epoch/node
-tar -xf /home/epoch/epoch.tar.gz -C /home/epoch/node
-chown -R epoch:epoch /home/epoch/node
+# Install ansible roles
+ansible-galaxy install -r requirements.yml
 
-cat > /home/epoch/node/epoch.yaml << EOF
----
-chain:
-    persist: true
+# Create temporary inventory just because of the group_vars
+cat > /tmp/local_inventory << EOF
+[local]
+localhost ansible_connection=local
 
-mining:
-    autostart: true
-    beneficiary: "${epoch_beneficiary}"
+[tag_role_epoch:children]
+local
 
-logging:
-    level: warning
+[tag_env_${env}:children]
+local
 
 EOF
 
-sudo su -c "/home/epoch/node/bin/epoch start" epoch
+ansible-playbook \
```

Comment:
I understand lines below couple instances with current (master - above) infrastructure repo. It means old images may not work though expected (and parallel PR shall cleanup old images - I understand).

Reply:
Yes, the old images will not be used once this is merged. And eventually cleanup.